lib/pushmi_pullyu/cli.rb

```diff
@@ -120,14 +140,18 @@ def setup_log
   end
 
   def run_tick_loop
+    queue = PushmiPullyu::PreservationQueue.new(connection: { host: config.redis_host, port: config.redis_port },
```

Comment:
It could. I went back and forth, since it's never needed outside of this method so this scopes it tighter, but it would look cleaner in the init routines. Either way is good with me.